docs/source/models/causes/other_causes/index.rst

```diff
@@ -74,16 +74,21 @@ States Data
      - 
    * - C
      - disabilty weight
-     - all_cause_yld_rate - anemia_yld_rate - population_maternal_disorders_yld_rate + pregnancy_maternal_disorders_yld_rate
+     - all_cause_yld_rate - anemia_yld_rate - population_maternal_disorders_yld_rate 
      - See extended note below
 
 .. note::
 
   Note that we are assuming the YLD rate due to all causes other than maternal disorders (including YLDs due to anemia) do not significantly vary between the pregnant/postpartum not pregnant/postpartum populations. We do rescale the rate of maternal disorders from the general population of women of reproductive age to the rate among the pregnant/postpartum population in this equation. 
 
-  Note that we include YLDs due to maternal disorders in the disability weight of "background morbidity" here despite the fact that maternal disorders is included as a cause of disability within the nutrition optimization pregnancy simulation. This is because in the simulation, simulants acquire pre-COMO-adjusted maternal disorders YLDs over the course of a single timestep in which accumulation of YLDs due to all other causes besides maternal disorders is paused. Therefore, since maternal disorders YLDs are not accumulated during the remainder of the simulation when anemia YLDs are accumulated, anemia YLDs would not be adjusted for comorbidity with maternal disorders unless they were included in the disability weight for background morbidity, which will be present during the timesteps for which anemia YLDs are accumulated.
+  Note that we are excluding YLDs due to maternal disorders in YLDs due to background morbidity, despite that YLDs due to maternal disorders in the nutrition optimization model are modeled separately and are never comorbid with other conditions. This limits us in that we therefore do not capture:
 
-    Note that while this allows us to more accurately estimate YLDs averted between scenarios, it will cause double counting of YLDs due to maternal disorders in the total amount of YLDs accrued within the simulation (due to all modeled causes combined).
+    1. any increase in maternal disorders YLDs due to reductions in anemia YLDs through the COMO adjustment,
+    2. any increase in anemia YLDs due to reductions in maternal disorders YLDs through the COMO adjustment, 
+    3. any increase in background morbidity YLDs due to reductions in maternal disorders, nor
+    4. COMO adjustments in YLDs due to anemia and other causes from comorbidity with maternal disorders. 
```

Comment:
How is this different from (2)?

Reply:
I have been thinking of two separate but definitely related impacts of our application of the "COMO" adjustment, including:
- Reductions in baseline cause-specific YLD rates due to comorbidities
- Ripple effects of increased cause-specific YLD rates between scenarios when comorbid cause-specific YLDs are reduced between scenarios

I am not sure if this is a helpful distinction even when more clearly articulated. Do you think it's worth keeping these as separate concepts with more explanation or should I condense in the list?

Reply:
I think I might be starting to see what you're saying, but it's still a little fuzzy. Is this a list of 4 differences between the equation before and after this PR, between the equation after this PR and not including background morbidity at all, between the equation after this PR and using the same equation but with maternal disorders modeled in a more typical way spread out over time, or between the equation after this PR and explicitly modeling every GBD cause (the ideal comorbidity adjustment)? Now that I read it over again, I think it is the third one, but I didn't get that from my initial read of the sentences before this list.

If I'm right about that, I think the distinction you're talking about can be useful, but it would be good to follow each item all the way through to what effect it has on YLDs averted. And, I'm still not sure which is which in the list.